ktxspec.adoc

```diff
@@ -744,14 +744,48 @@ These 3 items combined form a _Data Format Descriptor_
 The full specification for this is found in the _Khronos Data
 Format Specification_ version 1.3 <<KDF13>>.
 
-If the _dfDescriptor_ describes an sRGB transfer function then
-`<<_vkformat,vkFormat>>` must be one of the _SRGB_ formats.
-
 The _dfDescriptor_ is partially expanded here in order to provide
 sufficient information for a KTX2 file to be parsed without having to
 refer to <<KDF13>>. It consists of one or more _Descriptor Blocks_
 (dfDescriptorBlock).
 
+The following restrictions must be obeyed when setting the fields of the
+DFD. The value of `<<_vkformat,vkFormat>>` and the image data must never
+contradict the DFD.
+
+* `color_model` must be `KHR_DF_MODEL_RGBSDA`, `KHR_DF_MODEL_YUVSDA`
+  or one of the block compressed color models listed in <<KDF13>> Section
+  5.6 or its successors, currently `KDF_DF_MODEL_BC1A` to
+  `KDF_DF_MODEL_PVRTC2`.
+* `color_primaries` may be any of the available values since conversion
+  to a display's primaries is relatively straightforward.
+
+[NOTE]
+====
+Specifications which refer to KTX2 as their base format may wish to
+restrict the `color_primaries` that can be used.
+
+`KHR_DF_PRIMARIES_BT709/SRGB` is recommended for low dynamic range,
+standard gamut images.
```

Comment:
I think that IBL use case may be an exception here, since it uses values outside of `0 .. 1` range with sRGB primaries.

Reply:
I will add this exception when I add info about RGB[DEM] formats. I'll make a note in issue #80 to remind me.